app/models/manageiq/providers/foreman/provider.rb

```diff
@@ -23,7 +23,7 @@ class ManageIQ::Providers::Foreman::Provider < ::Provider
   before_validation :ensure_managers
 
   validates :name, :presence => true, :uniqueness => true
-  validates :url,  :presence => true, :uniqueness => true
+  validates :url,  :presence => true
```

Comment:
@matthewd This seems to fix the rspec issue I was running into.  I don't know why `:url` as a virtual_column isn't working well.  Maybe it needs to define `:url=` as well?  Either way, I don't think the uniqueness makes sense.

```
$ rspec spec/models/manageiq/providers/foreman/provider_spec.rb:24

Failures:

  1) ManageIQ::Providers::Foreman::Provider#destroy will remove all child objects
     Failure/Error: provider = FactoryGirl.create(:provider_foreman, :zone => FactoryGirl.create(:zone))

     NoMethodError:
       undefined method `limit' for nil:NilClass
     # /home/bdunne/.gem/ruby/2.2.4/bundler/gems/rails-d6769f4d106e/activerecord/lib/active_record/validations/uniqueness.rb:71:in `build_relation'
     # /home/bdunne/.gem/ruby/2.2.4/bundler/gems/rails-d6769f4d106e/activerecord/lib/active_record/validations/uniqueness.rb:18:in `validate_each'
     # /home/bdunne/.gem/ruby/2.2.4/bundler/gems/rails-d6769f4d106e/activemodel/lib/active_model/validator.rb:151:in `block in validate'
     # /home/bdunne/.gem/ruby/2.2.4/bundler/gems/rails-d6769f4d106e/activemodel/lib/active_model/validator.rb:148:in `each'
     # /home/bdunne/.gem/ruby/2.2.4/bundler/gems/rails-d6769f4d106e/activemodel/lib/active_model/validator.rb:148:in `validate'
     # /home/bdunne/.gem/ruby/2.2.4/bundler/gems/rails-d6769f4d106e/activesupport/lib/active_support/callbacks.rb:405:in `public_send'
     # /home/bdunne/.gem/ruby/2.2.4/bundler/gems/rails-d6769f4d106e/activesupport/lib/active_support/callbacks.rb:405:in `block in make_lambda'
     # /home/bdunne/.gem/ruby/2.2.4/bundler/gems/rails-d6769f4d106e/activesupport/lib/active_support/callbacks.rb:169:in `call'
```